src/js/services/werewolf.js

```diff
@@ -42,4 +42,6 @@ export default new class werewolfService {
     getScript(deck, lang = 'en') { //TODO: better defaults
         return brain.getScriptFromDeck(deck, lang);
     }
+
+    mode = brain.getModes;
```

Comment:
Line 46 is exporting `getModes` from the brain as part of the service. Did you mean to import `werewolf-brain` and use the modes from there?